examples/api/power_norm_demo.py

```diff
@@ -3,24 +3,23 @@
 import numpy as np
 from numpy.random import multivariate_normal
 
-data = np.vstack([multivariate_normal([10, 10], [[3, 2], [2, 3]], size=100000),
-                  multivariate_normal([30, 20], [[2, 3], [1, 3]], size=1000)
-                  ])
+data = np.vstack([
+    multivariate_normal([10, 10], [[3, 2], [2, 3]], size=100000),
+    multivariate_normal([30, 20], [[2, 3], [1, 3]], size=1000)
+])
 
 gammas = [0.8, 0.5, 0.3]
-xgrid = np.floor((len(gammas) + 1.) / 2)
```

Comment:
Agreed! As much as I appreciate clever ways to place axes, it's my opinion
that the examples should show off the smart ways to use `subplots` and
`gridspec`.
On Thu, Sep 8, 2016 at 18:45 Elliott Sales de Andrade <
notifications@github.com> wrote:

> In examples/api/power_norm_demo.py
> <https://github.com/matplotlib/matplotlib/pull/7067#discussion_r78118307>:
>
> >
> >  gammas = [0.8, 0.5, 0.3]
> > -xgrid = np.floor((len(gammas) + 1.) / 2)
>
> Whoa, really? That was unnecessarily confusing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/7067/files/182694fce5b1a78eb119d92772145b1b94a751a3#r78118307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHCo90lvE6McNphKazUnd28VNOkfaXIks5qoLo0gaJpZM4J4nIw>
> .
>